pkg/trace/info/writer.go

```diff
@@ -25,7 +25,6 @@ type TraceWriterInfo struct {
 	Retries           atomic.Int64
 	Bytes             atomic.Int64
 	BytesUncompressed atomic.Int64
```

Comment:
`BytesUncompressed` was only there to compare against `BytesEstimated` and be able to tell the accuracy of it. Do you think it's still useful going forward? The only thing we could gain from it is to analyse the efficiency of the compression. If not, maybe we can remove that too.

Reply:
I see value in this metric both to monitor compression efficiency (more especially as we'll work on improving it soon) and as a way to estimate uncompressed ingestion (which is what's measured for span ingestion billing) per host.